src/qt/pivx/sendconfirmdialog.cpp

```diff
@@ -128,7 +128,10 @@ void TxDetailDialog::setData(WalletModel *model, WalletModelTransaction &tx)
     ui->textAmount->setText(BitcoinUnits::formatWithUnit(nDisplayUnit, totalAmount, false, BitcoinUnits::separatorAlways) + " (Fee included)");
     int nRecipients = tx.getRecipients().size();
     if (nRecipients == 1) {
-        SendCoinsRecipient recipient = tx.getRecipients().at(0);
+        const SendCoinsRecipient& recipient = tx.getRecipients().at(0);
+        if (recipient.isP2CS) {
+            ui->labelSend->setText("Delegating to");
```

Comment:
missing translation `tr("Delegating to")`.

Reply:
fixed